lms/services/lti_h.py

```diff
@@ -1,17 +1,39 @@
+from functools import wraps
```

Comment:
This whole file looks horrible in the diff, but if you read the plain code it's miles more clear